UnitsNet.Tests/CustomCode/VolumeFlowTests.cs

```diff
@@ -96,6 +96,8 @@ public class VolumeFlowTests : VolumeFlowTestsBase
 
         protected override double OilBarrelsPerMinuteInOneCubicMeterPerSecond => 3.77388865E2;
 
+        protected override double KilousGallonsPerMinuteInOneCubicMeterPerSecond => 15850.323141489e-3;
```

Comment:
Correct number, but please use scientific notation: `1.5850323141489e1`. Verified with [source](https://www.unitjuggler.com/convert-flowrate-from-m3s-to-us-gpm.html).

Reply:
Fixed with latest commit